scripts/diagnose_recordings.py

```diff
@@ -0,0 +1,370 @@
+#!/usr/bin/env python3
```

Comment:
cc @slekkala1 @ehhuang @leseb @derekhiggins as you all battle with the recording system :)